### PR TITLE
feat: FTL-16681 add commands to manage credential issuance configuration

### PR DESCRIPTION
**Pull request by rbrazhnyk**

### CODE REVIEW

**Review by Bergmam — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -136,6 +137,9 @@
       },
       "wallet": {
         "description": "Use these commands to manage your wallets"
+      },
+      "issuance": {
+        "description": "Use these commands to manage credential issuance configuration"
```
Comment (carlos-affinidi) [inline package.json:142]:
```suggestion
        "description": "Use these commands to manage credential issuance configurations"
```

File: src/commands/issuance/create-config.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      summary: 'ID of the wallet',
+    }),
+    'credential-offer-duration': Flags.integer({
+      summary: 'Credential offer duration in seconds',
+    }),
+    file: Flags.string({
+      char: 'f',
+      summary:
+        'Location of a json file containing the list of allowed schemas for creating a credential offer. One or more schemas can be added to the issuance. The credential type ID must be unique',
+    }),
```
Comment (carlos-affinidi) [inline src/commands/issuance/create-config.ts:45]: How can the developer figure out the structure of this file?
I think we will end up requiring separate CRUD commands for credential configs as we are now also adding metadata display fields to these.

File: src/commands/issuance/create-config.ts [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      const walletChoices =
+        wallets?.map((wallet: WalletDto) => ({
+          name: wallet.name || wallet.id,
+          value: wallet.id,
+        })) || []
+
+      const CREATE_NEW_WALLET = 'Create new wallet'
+      walletChoices.push({ name: CREATE_NEW_WALLET, value: CREATE_NEW_WALLET })
+
+      const selectedWallet = await select({
```
Comment (carlos-affinidi) [inline src/commands/issuance/create-config.ts:81]: Check no-input flag before prompting

**Review by carlos-affinidi — CHANGES_REQUESTED** [review]

File: src/commands/issuance/create-config.ts (hunk 3 above)
Comment (rbrazhnyk) [reply src/commands/issuance/create-config.ts:81]: The check happens [here](https://github.com/affinidi/affinidi-cli/pull/400/files/c3f412cdd6ee3768e5eda7dcca3a402a3dfe1fc7#diff-3f20c6f3e1210b31064248fb4e17f262614fbceb146be1be39ff33bc1485d58bR51) - and error will be thrown if no value provided

File: src/commands/issuance/create-config.ts (hunk 2 above)
Comment (rbrazhnyk) [reply src/commands/issuance/create-config.ts:45]: That is a good point! We might need to cover it for all places where "file" is an input.

File: src/commands/issuance/create-config.ts (hunk 3 above)
Comment (rbrazhnyk) [reply src/commands/issuance/create-config.ts:81]: Screenshot 2024-09-02 at 20 09 50

**Review by carlos-affinidi — APPROVED** [review]

**Pull request merged**